app/src/main/java/mozilla/lockbox/presenter/ItemMutationPresenter.kt

```diff
@@ -65,7 +65,11 @@ abstract class ItemMutationPresenter(
         view.saveEntryClicks
             .flatMap { itemDetailStore.isDirty.take(1) }
             .map { saveChangesAction(it) }
-            .subscribe(dispatcher::dispatch)
+            .subscribe { itemDetailActions ->
+                itemDetailActions.filterNotNull().forEach { action ->
+                    dispatcher.dispatch(action)
+                }
+            }
```

Comment:
If `saveChangesAction()` returns a `List<Action>`, then this could be simpler:

```kotlin
.flatMapIterable { saveChangesAction(it) }
.subscribe(dispatcher::dispatch)
```